app/src/main/java/com/example/yummy/ui/main/MainActivity.kt

```diff
@@ -0,0 +1,39 @@
+package com.example.yummy.ui.main
+
+import android.content.Context
+import android.content.Intent
+import com.example.yummy.R
+import com.example.yummy.base.BaseActivity
+import com.example.yummy.ui.favorite.FavoriteFragment
+import com.example.yummy.ui.home.HomeFragment
+import com.example.yummy.ui.news.NewsFragment
+import com.example.yummy.ui.note.NoteFragment
+import com.example.yummy.ui.search.AreaFragment
+import com.google.android.material.bottomnavigation.BottomNavigationView
+import kotlinx.android.synthetic.main.activity_main.*
+
+class MainActivity : BaseActivity() {
+    override val layoutResource: Int
+        get() = R.layout.activity_main
```

Comment:
Tại sao không viết như này nhỉ, có gì khác nhau không ? 
```suggestion
    override val layoutResource: Int = R.layout.activity_main
```

Reply:
:) ừ a

Reply:
nếu bỏ get() đi thì có thể gán giá trị là layout, drawable, ...
Còn nếu viết như ban đầu thì phải gán giá trị là layout ạ